src/app/market/listings/preview-listing/preview-listing.component.html

```diff
@@ -41,10 +41,11 @@
           Local in {{ data?.listing?.country }} &ndash; {{ data?.listing?.domesticShippingPrice.getAmount() }} PART
         </td>
       </tr>
-      <tr>
+      <!-- @TODO how we calculate escrow as we don't know the shipping country -->
```

Comment:
That's a very good question. 
How do know which country price to pick IF we only get the addresses in the second step of the checkout.
Bring it up with the UX guys, @AllienWorks

Reply:
We had a discussion around this question on the group a couple days back. @xludx mentioned if there has been a previous order, we might be able to get this to work.